api/src/pcapi/core/offerers/api.py

```diff
@@ -1174,11 +1174,7 @@ def can_offerer_create_educational_offer(offerer_id: int) -> None:
 def can_venue_create_educational_offer(venue_id: int) -> None:
     import pcapi.core.educational.adage_backends as adage_client
 
-    offerer = (
-        offerers_models.Offerer.query.join(offerers_models.Venue, offerers_models.Offerer.managedVenues)
-        .filter(offerers_models.Venue.id == venue_id)
-        .one()
-    )
+    offerer = offerers_models.Venue.query.get(venue_id).managingOfferer.one()
```

Comment:
Plutôt qu’une exception `NoResultFound`, on risque d’avoir maintenant cette erreur:

```python
AttributeError: 'NoneType' object has no attribute 'managingOfferer'
```

Par ailleurs, `one()` n’est pas nécessaire,

Pourquoi pas juste
```suggestion
    offerer = (
        offerers_models.Offerer.query.join(offerers_models.Venue, offerers_models.Offerer.managedVenues)
        .filter(offerers_models.Venue.id == venue_id)
        .one_or_none()
    )
    if offerer is None:
       # faire quelque chose de cohérent dans le contexte
       # retourner une erreur, etc...
```

Reply:
Et le bug ne se produit pas seulement quand l’id de venue vaut 0, simplement quand c’est un id de venue qui n’est pas présent en db

Reply:
Merci pour les informations j'avais pas du tout compris ca :)